app/forms/teacher_interface/new_session_form.rb

```diff
@@ -0,0 +1,14 @@
+class TeacherInterface::NewSessionForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :email, :string
+  attribute :create_or_sign_in, :string
+
+  validates :create_or_sign_in, presence: true
+  validates :email, presence: true, valid_for_notify: true, if: -> { sign_in? }
```

Comment:
I think we're missing a `sign_in?` method here (we're using one but it's not defined), I suggested in https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/366#discussion_r947512089 it would like something like:

```rb
def sign_in?
  create_or_sign_in == "sign_in"
end
```